core-shared-ui/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,10 @@
 <resources>
     <string name="app_name">StreamPlayerApp</string>
```

Comment:
por que tu criou dois arquivos de strings.xml?

Reply:
Pra separar as strings do content-description

Como o compose meio q força a gente a trabalhar mais com acessibilidade, principalmente o talk back, a gente precisa setar o valor do content-description pras imagens. 

A gente poderia deixar tudo num arquivo tb, mas eu tenho separado e tenho gostado da organização. 

Faz sentido?